geniza/settings/components/base.py

```diff
@@ -64,7 +64,6 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [BASE_DIR / 'geniza' / 'templates'],
```

Comment:
This change lost the top-level admin index and login override templates (for pucas login). Maybe they are in the wrong location and should be moved, but we need them included somewhere.